CHANGELOG-1.6.md

```diff
@@ -0,0 +1,21 @@
+# Changes since v1.5.0
```

Comment:
I should probably add deprecation notices for everything we plan to break with 2.0